### DISCUSSION TITLE
Are contributions welcome or not?

### DISCUSSION THREAD

**Discussion by kagahd** [DefectDojo/django-DefectDojo#10487] [General]
I wonder whether contributions are welcome or not because more than three weeks ago I opened PR #10372 which has already 3 approvals but got not merged yet. Therefore, merge conflicts will accumulate and will be harder to resolve. Hence my question, DefectDojo team, are you really interested in contributions or not?

**Comment by mtesauro** [comment]
Repeating what I said [here](https://github.com/DefectDojo/django-DefectDojo/pull/10372#issuecomment-2204438647):

Yes, we're always happy to have PRs from the community. It's summer time (at least in the northern hemisphere) and the combination of:
* several maintainers going on holiday recently
* several large PRs that were queued up for 2.36.0 release (which happened yesterday)
* preparation for deprecation of MySQL and RabbitMQ (see [here](https://github.com/DefectDojo/django-DefectDojo/discussions?discussions_q=author%3Amtesauro) especially [this one](https://github.com/DefectDojo/django-DefectDojo/discussions/9690))
* something strange happening with Github caches for GH Actions not being correctly evicted in a timely manner and slowing down our GH Action runs
* I was at OWASP's AppSec Lisbon presenting on DefectDojo last week so am just now catching up on PRs myself

has kept the maintainers busier that normal in June.

I'm sorry you had to fix several merge conflicts but this is a very active open source project which a code base that changes quickly since the community is quite active.  We (the maintainers) do our best to keep PR open as short as possible but things don't work out as planned in every situation. :+1:
